bin/generate-apidocs.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
+var fs = require('fs');
+var path = require('path');
+var tsPath;
+try {
+  tsPath = require.resolve('typedoc/node_modules/typescript/package.json');
+} catch (e) {
+  // Ignore the error
+}
+if (tsPath) {
+  tsPath = path.resolve(tsPath, '..');
+  // Remove the nested typescript module for typedoc so that it uses the
+  // typescript version for our projects
+  try {
+    fs.renameSync(tsPath, tsPath + '.bak');
```

Comment:
Is this mostly to avoid needing a library like `rimraf` to recursively remove the directory?

Reply:
Yes